lib/generators/ruby_ui/install/templates/application.css.erb

```diff
@@ -0,0 +1,153 @@
+@import "tailwindcss";
+@import "../../../vendor/javascript/tw-animate-css.js";
```

Comment:
What is this line? I think something from your project might have snuck in here...?

Reply:
Or maybe this is somehow related to the tailwind animate plugin? But when I ran `importmap pin tailwindcss-animate` it did not add that file to my vendor dir:
```
importmap pin tailwindcss-animate
Pinning "tailwindcss-animate" to vendor/javascript/tailwindcss-animate.js via download from https://ga.jspm.io/npm:tailwindcss-animate@1.0.7/index.js
Pinning "tailwindcss/plugin" to vendor/javascript/tailwindcss/plugin.js via download from https://ga.jspm.io/npm:tailwindcss@4.0.14/dist/plugin.js
```
Weirdly the output seems to indicate that it saves the tailwind plugin file at `vendor/javascript/tailwindcss/plugin.js` but instead it added `vendor/javascript/tailwindcss--plugin.js`